uptane/common.py

```diff
@@ -247,8 +249,8 @@ def sign_over_metadata(
         {'signed': data, 'signatures': []}, only_signed=True, datatype=datatype)
     data = hashlib.sha256(data).digest()
 
-  else:
-    raise tuf.Error('Unsupported metadata format: ' + repr(metadata_format))
+  else: # pragma no cover
+    raise uptane.Error('Unsupported metadata format: ' + repr(metadata_format))
```

Comment:
Consider printing the metadata formats that *are* supported.  The same also for unsupported key types (i.e., SUPPORTED_KEY_TYPES).